ffmpeg/ffmpeg.go

```diff
@@ -68,6 +68,9 @@ type TranscodeOptions struct {
 type MediaInfo struct {
 	Frames int
 	Pixels int64
+	//for dnn result
+	Isdnn     bool
+	Dnnresult string
```

Comment:
Can be changed to `IsDNN` and `DNNResult` to be consistent with naming of other variables.